src/components/TokenIcon.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="token-icon">
+    <img
+      v-if="tokenA"
+      :class="{ rotating, 'not-pool-token': !tokenB }"
+      :src="mapToken(tokenA)"
+    >
+    <img
+      v-if="tokenB"
+      :class="{ rotating }"
+      :src="mapToken(tokenB)"
+    >
+  </div>
+</template>
+
+<script>
+import AeIcon from '../assets/ae.svg';
+
+export default {
+  props: {
+    tokenA: { type: Object, required: true },
+    tokenB: { type: Object, default: null },
+    rotating: { type: Boolean },
+  },
+  methods: {
+    mapToken(token) {
+      return token.is_ae !== undefined ? AeIcon : `https://avatars.z52da5wt.xyz/${token.contract_id}`;
```

Comment:
In my case, when used from PoolView, `token.is_ae` is always `null`

<img width="500" alt="image" src="https://user-images.githubusercontent.com/3176255/178331814-3da3d5b6-7fab-4d31-a907-322285d34ee7.png">

Reply:
Shouldn't tokens have the same appearance everywhere on all modals?

Reply:
fixed